### DISCUSSION TITLE
Unable to bundle for desktop application

### DISCUSSION THREAD

**Discussion by rjimaw7** [alex8088/electron-vite#83] [General]
Hello I'm new to using the electron vite . I Followed the steps on the documentation :

Steps that I did : 

- yarn create @quick-start/electron my-app --template react
- cd my-app
- yarn
- yarn dev 

The application starts running successfully . Then I wanted to test to bundle to a desktop app . I tried this command 

- yarn build:win

Then I got the error on the screenshot attached . Please help me . Thank you .

![unbale](https://user-images.githubusercontent.com/42797332/210766511-2abd6ff3-30ce-48fe-9544-d9426cdb00e4.PNG)

**Comment by alex8088** [comment]
May be caused by  anti-virus，try more
